### DISCUSSION TITLE
[kubernetes] Add `json` as Log output format

### DISCUSSION THREAD

**Discussion by nlamirault** [airbytehq/airbyte#36312] [Connector Ideas and Features]
### Page URL

https://docs.airbyte.com/operator-guides/configuring-airbyte#logging

### Description

I use Airbyte in a Kubernetes cluster. I would like JSON format for Airbyte logs. Is it possible to change the output format of logs?

**Comment by marcosmarxm** [comment]
Today it isn't possible to change the log format. I'm moving this request to a Github Discussion to get more feedback from other users and further discussions.

[a comment by an automated account is not shown]
